### ISSUE TITLE
ClinVar variant missing

### ISSUE THREAD

**Issue by AlistairNWard** [tonydisera/gene.iobio#624]
Open demo data in RAI1. There are 4 variants, 3 of which are in ClinVar. I think all the variants are actually in ClinVar, including the deletion that is not marked as such.

**Comment by tonydisera** [comment]
Good catch, Al!  I agree that the deletion should be showing.  Here is the clinvar entry:

https://www.ncbi.nlm.nih.gov/clinvar/variation/96187/

So now I will track down why this isn't getting returned from eutils.  Interestingly, the dbSNP entry doesn't have a link to clinvar either for the deletion, but does have a link for the other 3 variants in RAI1.

<img width="700" alt="screen shot 2017-06-08 at 4 39 37 pm" src="https://user-images.githubusercontent.com/8420103/26953691-2305ea12-4c69-11e7-8298-2b83f6784d25.png">

https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs149716029
<img width="365" alt="screen shot 2017-06-08 at 4 39 10 pm" src="https://user-images.githubusercontent.com/8420103/26953659-06c601e8-4c69-11e7-899b-2300b8c9b420.png">

**Comment by tonydisera** [comment]
It is interesting that the variant entry int dbSNP doesn't have a link to clinvar either for the deletion, but does have a link for the other 3 variants in RAI1.


https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs149716029
<img width="365" alt="screen shot 2017-06-08 at 4 39 10 pm" src="https://user-images.githubusercontent.com/8420103/26953659-06c601e8-4c69-11e7-899b-2300b8c9b420.png">

**Comment by tonydisera** [comment]
It looks like the problem is the reported position in ClinVar for GRCh37.

The deletion is reported in position 17:17700037 in the VCF:

<img width="658" alt="screen shot 2017-06-08 at 4 48 11 pm" src="https://user-images.githubusercontent.com/8420103/26953882-4df81942-4c6a-11e7-9c66-457936bcdf96.png">

And position 17:17700038 in dbSNP, which is what we expect (deletions are +1 from VCF position):

<img width="380" alt="screen shot 2017-06-08 at 4 49 18 pm" src="https://user-images.githubusercontent.com/8420103/26953912-74e332f8-4c6a-11e7-84fe-a5439832e33d.png">

**Comment by AlistairNWard** [comment]
The coordinate should be to the leftmost flanking base, e.g. that base that is present regardless of whether the deletion is present or not. So it seems that 17700037 should be the reported coordinate. The clinVar vcf that I have has the deletion at this location. Very odd.

**Comment by tonydisera** [comment]
Thanks Al.  I did a quick validation that other deletions are associated with the clinvar annotations by pulling up the clinvar vcf in gene.  So I think this might be a mistake in clinvar.
